unity-renderer/Packages/manifest.json

```diff
@@ -33,7 +33,7 @@
     "com.unity.xr.legacyinputhelpers": "2.1.10",
     "io.sentry.unity": "https://github.com/getsentry/unity.git#0.28.0",
     "net.tnrd.nsubstitute": "https://github.com/decentraland/Unity3D-NSubstitute.git",
-    "ru.threedisevenzeror.unitygifdecoder": "https://github.com/3DI70R/Unity-GifDecoder.git",
+    "ru.threedisevenzeror.unitygifdecoder": "https://github.com/decentraland/Unity-GifDecoder.git#fix/fallback-for-non-power-of-2",
```

Comment:
Remember to remove the branch after merging the PR :D

Reply:
Done :D